doc/sphinx/Clusters_from_Scratch/fencing.rst

```diff
@@ -151,40 +151,39 @@ Step 4: Obtain the agent's possible parameters:
-                      the sum is kept below the maximum delay.
-      pcmk_delay_base: Enable a base delay for stonith actions and specify base delay value. This prevents double
-                       fencing when different delays are configured on the nodes. Use this to enable a static delay for
-                       stonith actions. The overall delay is derived from a random delay value adding this static delay
-                       so that the sum is kept below the maximum delay.
-      pcmk_action_limit: The maximum number of actions can be performed in parallel on this device Cluster property
-                         concurrent-fencing=true needs to be configured first. Then use this to specify the maximum
-                         number of actions can be performed in parallel on this device. -1 is unlimited.
-
-     Default operations:
-       monitor: interval=60s
+      pcmk_host_check: How to determine which machines are controlled by the device. Allowed values: dynamic-list (query the device via the
+                       'list' command), static-list (check the pcmk_host_list attribute), status (query the device via the 'status' command),
+                       none (assume every device can fence every machine)
+      pcmk_delay_max: Enable a random delay for stonith actions and specify the maximum of random delay. This prevents double fencing when
+                      using slow devices such as sbd. Use this to enable a random delay for stonith actions. The overall delay is derived from
+                      this random delay value adding a static delay so that the sum is kept below the maximum delay.
+      pcmk_delay_base: Enable a base delay for stonith actions and specify base delay value. This prevents double fencing when different delays
+                       are configured on the nodes. Use this to enable a static delay for stonith actions. The overall delay is derived from a
+                       random delay value adding this static delay so that the sum is kept below the maximum delay.
+      pcmk_action_limit: The maximum number of actions can be performed in parallel on this device Cluster property concurrent-fencing=true
+                         needs to be configured first. Then use this to specify the maximum number of actions can be performed in parallel on
+                         this device. -1 is unlimited.
+
+    Default operations:
+      monitor: interval=60s
 
 Step 5: ``pcs cluster cib stonith_cfg``
 
 Step 6: Here are example parameters for creating our fence device resource:
 
 .. code-block:: none
 
-    [root@pcmk-1 ~]# pcs -f stonith_cfg stonith create my_stonith fence_virt \
-          pcmk_host_list="pcmk-1 pcmk-2" ipaddr=10.0.0.1 op monitor interval=60s
+    [root@pcmk-1 ~]# pcs -f stonith_cfg stonith create ipmi-fencing fence_ipmilan \
+          pcmk_host_list="pcmk-1 pcmk-2" ipaddr=10.0.0.1 login=testuser \
+          passwd=acd123 op monitor interval=60s
```

Comment:
This is perfectly fine, no need to change, but just FYI, the name of the device (my_stonith vs ipmi-fencing) is arbitrary, not tied to the type of device used, so it would also have been fine to keep my_stonith as the device name.